tools/gulptasks/scripts-webpack.js

```diff
@@ -34,6 +167,15 @@ async function scriptsWebpack() {
         configs = {
             Grid: 'grid.webpack.mjs'
         };
+    } else if (argv.dashboards) {
+        configs = {
+            Dashboards: 'dashboards.webpack.mjs'
+        };
```

Comment:
We can We can completely remove the `--dashboards` arg in favor of `--product Dashboards` (check globally)

Reply:
There are still some places with `argv.dashboards` that could be removed (`lint-ts.js`, `scripts-css.js`, `test-karma.js`)
<img width="483" height="129" alt="image" src="https://github.com/user-attachments/assets/f884f5d3-481c-4ea5-b16c-18ad0832d63f" />